README.md

```diff
@@ -124,3 +129,51 @@ A student list / progress tracker is available for admins at
 http://<IPADDRESS>/users/list.php
 ```
 You are required to enter your Domain Admin password to login to see the list.
+
+## Deleting the environment
+
+Run the `teardown.yml` playbook to tear down the environment. The playbook deletes the instances, the VPCs, and any other resource that the installer creates.
+It takes roughly five minutes for this playbook to run.
+
+```
+ansible-playbook teardown.yml
```

Comment:
We might want to make a note that if you have multiple workshops running at the same time, then you should point the vars file for the workshop you want to teardown.  Such as 
ansible-playbook teardown -e @vars/AnsibleAutomatesDallas.yml
I have had up to 3 workshops provisioned at once, so its a good habit to get into.